hw2/04-background.js

```diff
@@ -1 +1,58 @@
-// Add your code here
+const button = document.querySelector("#start"); // start button click
+button.addEventListener("click", handleClick);
+
+const userInput = document.querySelector("#num"); // num seconds
+window.addEventListener("load", handleLoad);
+
+// Setup
+let intervalID;
+
+// Create an array of colors to select from
+const colors = [
+  "lightcoral",
+  "cadetblue",
+  "darkolivegreen",
+  "darkmagenta",
+  "darkgoldenrod",
+  "darkslateblue",
+  "cornflowerblue",
+];
```

Comment:
I'd double check, but was under the impression it had to be a random color in general, not from a set. I would think this is acceptable though, as the other approach is actually really challenging.

Reply:
I emailed Caterina about it.  Will update!

Reply:
I heard back from Caterina and she said my approach was okay!